.github/workflows/ci.yml

```
@@ -15,7 +15,7 @@ jobs:
     timeout-minutes: 30
     strategy:
       matrix:
-        os: [macOS-latest, ubuntu-latest, windows-2019]
+        os: [macOS-latest, ubuntu-latest, windows-2022]
```

```suggestion
        os: [macOS-latest, ubuntu-latest, windows-latest]
```

Let's just do latest